servicex/minio_adapter.py

```diff
@@ -28,22 +28,25 @@
 import os.path
 from hashlib import sha1
 from pathlib import Path
-from typing import List
+from typing import List, Optional
 
 from tenacity import retry, stop_after_attempt, wait_random_exponential
 
 import aioboto3
 from boto3.s3.transfer import TransferConfig
+import asyncio
 
 from servicex.models import ResultFile, TransformStatus
 
-_transferconfig = TransferConfig(max_concurrency=10)
+_transferconfig = TransferConfig(max_concurrency=5)
+_sem = asyncio.Semaphore(10)
```

Comment:
These important magic numbers need  more comments to highlight their funciton. I also think `_sem` is not a great variable name

Reply:
Renamed and comments added